css/main.css

```diff
@@ -307,21 +307,134 @@ li {
 }
 
 .list-title {
-  font-family: "Inter", sans-serif;
+  font-family: 'Inter', sans-serif;
   font-style: normal;
   font-weight: bold;
   font-size: 32px;
   line-height: 44px;
 }
 
 .languages {
-  list-style: inside url("../media/svg/rectangle.svg");
+  list-style: inside url('../media/svg/rectangle.svg');
 }
 
 .frameworks {
-  list-style: inside url("../media/svg/square.svg");
+  list-style: inside url('../media/svg/square.svg');
 }
 
 .skills {
-  list-style: inside url("../media/svg/circle.svg");
+  list-style: inside url('../media/svg/circle.svg');
+}
+
+.contact-section {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  margin-bottom: 50px;
+}
+
+.contact-section-head {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
+
+.contact-section-info {
+  margin-top: 50px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  font-family: Arial, Helvetica, sans-serif;
+  text-align: center;
+  width: 327px;
+  color: #172b4d;
+}
+
+.contact-form {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.fieldset {
+  padding: 0;
+  margin: 0;
+  border: none;
+}
+
+.contact-form input {
+  margin-bottom: 20px;
+  border: none;
+  border-bottom: 1px solid #cfd8dc;
+  outline: none;
+  background-color: white;
+  height: 50px;
+  font-size: 18px;
+  color: #172b4d;
+  padding-left: 22px;
+  width: 90%;
+}
+
+.textarea::placeholder {
+  color: #3c3a39;
+  font-weight: 500;
+}
+
+.textarea {
+  border: 1px solid #cfd8dc;
+  margin-bottom: 20px;
+  width: 90%;
+  height: 170px;
+  padding-left: 22px;
+  padding-top: 25px;
+  font-size: 18px;
+  outline: none;
+  background-color: #fbf8f7;
+}
+
+.contact-form input:focus,
+.contact-form textarea:focus {
+  border-color: #000;
+}
+
+.submit-contact {
+  text-align: center;
+  width: 140px;
+  height: 48px;
+  background: #ff6b00;
+  border: none;
+  font-size: 20px;
+  color: white;
+  cursor: pointer;
+  transition: all 0.5s;
+}
+
+.submit-contact:hover {
+  color: #ff6b00;
+  background-color: white;
+  border: 2px solid #ff6b00;
+}
+
+.contact-link {
+  color: #505f79;
+  margin-top: 50px;
+  text-align: center;
+}
+
+.contact-icon {
+  color: #505f79;
+}
```

Comment:
- [ ] Kindly, make sure you stick to the the provide design. Make sure you social link icon have the same as the one in the figma design.

     - Your design 
  
   ![image](https://user-images.githubusercontent.com/87197412/184038335-74804865-6805-448f-83e8-3d5d8e9a1ac6.png)
 
   - Figma design
  
  ![image](https://user-images.githubusercontent.com/87197412/184038428-d6591420-e4aa-4a0c-836a-866320c717a0.png)


- [ ] Please make sure to add a horizontal line which comes before the icon container.

     - Your design 
      
   ![image](https://user-images.githubusercontent.com/87197412/184038849-4cb80acb-7510-4128-ab32-18be655c63e1.png)

    - Figma design
         
![image](https://user-images.githubusercontent.com/87197412/184038768-891e5173-1b46-459e-9caa-c3eac480a616.png)